migrations/versions/f3c7a489a0c0_removed_active_column_in_eligibility_.py

```diff
@@ -28,8 +28,7 @@ def upgrade():
 def downgrade():
     # ### commands auto generated by Alembic - please adjust! ###
     op.add_column('eligibility_criteria_info', sa.Column('active', sa.BOOLEAN(), autoincrement=False, nullable=True))
-    op.drop_column('eligibility_criteria_info', 'status')
```

Comment:
I would avoid editing an already existing migration file. This will create some issues in the migrations since this had already been executed in the various environments so this change will not be picked up.

Reply:
Yes, I'll take a look

Reply:
This is fixed. We're good to go for this PR.